dcpy/connectors/esri/arcgis_feature_service.py

```diff
@@ -36,10 +36,10 @@ def get_feature_server_layers(
         FeatureServerLayer(
             server=feature_server.server,
             name=feature_server.name,
-            layer_name=l["name"],
```

Comment:
pep8 doesn't like variables named `l` because of its visual similarity to `1` or `I`